src/System.Security.Cryptography.Cng/tests/Configurations.props

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netfx-Windows_NT;
```

Comment:
Ok so I tried this but it turns out we can't. We still depend on the System.Security.Cryptography.Algorithms platform specific contract, since DSACng type inherits from DSA class (defined in Algorithms) which netstandard.dll surface area is missing the HashData() members. These members are only present in the corefx contracts, but netstandard.dll is missing them. Two ways to go with Cng then, one is to add these HashData methods to netstandard, or if not then keep it as is with having all ref, impl and tests as platform-specific.